docs/API/Elements.md

```diff
@@ -343,6 +381,10 @@ function enter() {
 }
 ```
 
+If you would also like to set a default name for the child in the inspector, for example <i>onObject</i> change the code:
```

Comment:
I think maybe "...for the child in the inspector (for example, _onObject_), change the code to:" flows a little bit better